co-simulation/lib/mosaic-objects/src/main/java/org/eclipse/mosaic/lib/objects/detector/Size.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (C) 2023 LEIDOS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.eclipse.mosaic.lib.objects.detector;
+
+import java.io.Serializable;
+
+public class Size implements Serializable {
+    private static final long serialVersionUID = 1L;
+
+    private double length;
+
```

Comment:
JavaDoc comment for each member

Reply:
As mentioned before I do not think a Javadoc comment for each member is required in Java where getters/setters already have that information about the member .